plugins/Marketplace/resources/qml/OnboardBanner.qml

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2021 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.1
+
+import UM 1.6 as UM
+import Cura 1.6 as Cura
+
+// Onboarding banner.
+Rectangle
+{
+    Layout.preferredHeight: childrenRect.height + 2 * UM.Theme.getSize("default_margin").height
+    anchors
+    {
+        margins: UM.Theme.getSize("default_margin").width
+        left: parent.left
+        right: parent.right
+        top: parent.top
+    }
+
+    color: UM.Theme.getColor("action_panel_secondary")
+
+    // Icon
+    Rectangle
```

Comment:
If an rectangle doesn't have a background or a border, it's better to use Item (since that doesn't need to be checked if it needs to be drawn)

Reply:
https://github.com/Ultimaker/Cura/pull/10948/commits/f1a9bbd7914b3dbdce17e0e2ec4d02b779c6acf1